### PR TITLE
Add work and about me with skill

### PR DESCRIPTION
**Pull request by tusinahmed**
In this pull request, I added about me section with works and skills.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -62,7 +62,256 @@ <h4 class="sec-headline">i'm a software developer</h4>
...
+          <div>
+            <ul class="allul">
+              <li>React.js</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
```
Comment (abdulhamiid) [inline index.html:290]: According to the project requirement, Please let's use `ul` and `li` for categories in the "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items. 💪🛠

Requirement screenshot 👇 

image

File: index.html [hunk 2]
```diff
@@ -62,7 +62,256 @@ <h4 class="sec-headline">i'm a software developer</h4>
...
+
+     
+
+      </section>
+
+       <!-- Work part start -->
+       <section class="works-part">
+        <div class="work">
+          <div class="work-image-part">image</div>
+
```
Comment (abdulhamiid) [inline index.html:80]: Figma
image

Your design
image

According to the project requirements, it is important we stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Please let's update the font face and use the same font as the Figma design. 👍 🤗

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @tusinahmed 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ UI Design
- ✔️ Highly organized code
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tusinahmed** [comment]
Add all updates which I get from reviews. update here

**Review by emmiiorji — APPROVED** [review]
Hi @tusinahmed, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**